sql/updates/reverse-dev.sql

```diff
@@ -152,3 +152,23 @@ GRANT SELECT ON _timescaledb_catalog.chunk TO PUBLIC;
 
 -- end recreate _timescaledb_catalog.chunk table --
 
+-- reverse cagg catalog relations
+DROP PROCEDURE @extschema@.cagg_migrate (REGCLASS, BOOLEAN, BOOLEAN);
+DROP FUNCTION _timescaledb_internal.cagg_migrate_pre_validation (TEXT, TEXT, TEXT);
+DROP PROCEDURE _timescaledb_internal.cagg_migrate_create_plan (_timescaledb_catalog.continuous_agg, TEXT, BOOLEAN, BOOLEAN);
+DROP FUNCTION _timescaledb_internal.cagg_migrate_plan_exists (INTEGER);
+DROP PROCEDURE _timescaledb_internal.cagg_migrate_execute_plan (_timescaledb_catalog.continuous_agg);
+DROP PROCEDURE _timescaledb_internal.cagg_migrate_execute_create_new_cagg (_timescaledb_catalog.continuous_agg, _timescaledb_catalog.continuous_agg_migrate_plan_step);
+DROP PROCEDURE _timescaledb_internal.cagg_migrate_execute_disable_policies (_timescaledb_catalog.continuous_agg, _timescaledb_catalog.continuous_agg_migrate_plan_step);
+DROP PROCEDURE _timescaledb_internal.cagg_migrate_execute_enable_policies (_timescaledb_catalog.continuous_agg, _timescaledb_catalog.continuous_agg_migrate_plan_step);
+DROP PROCEDURE _timescaledb_internal.cagg_migrate_execute_copy_policies (_timescaledb_catalog.continuous_agg, _timescaledb_catalog.continuous_agg_migrate_plan_step);
+DROP PROCEDURE _timescaledb_internal.cagg_migrate_execute_refresh_new_cagg (_timescaledb_catalog.continuous_agg, _timescaledb_catalog.continuous_agg_migrate_plan_step);
+DROP PROCEDURE _timescaledb_internal.cagg_migrate_execute_copy_data (_timescaledb_catalog.continuous_agg, _timescaledb_catalog.continuous_agg_migrate_plan_step);
+DROP PROCEDURE _timescaledb_internal.cagg_migrate_execute_override_cagg (_timescaledb_catalog.continuous_agg, _timescaledb_catalog.continuous_agg_migrate_plan_step);
+DROP PROCEDURE _timescaledb_internal.cagg_migrate_execute_drop_old_cagg (_timescaledb_catalog.continuous_agg, _timescaledb_catalog.continuous_agg_migrate_plan_step);
+
+ALTER EXTENSION timescaledb DROP SEQUENCE _timescaledb_catalog.continuous_agg_migrate_plan_step_step_id_seq;
```

Comment:
```suggestion
ALTER EXTENSION timescaledb DROP SEQUENCE pg_get_serial_sequence('_timescaledb_catalog.continuous_agg_migrate_plan_step', 'step_id');
```
Do we need to drop the serial if we drop the table? We probably don't need to use `pg_get_serial_sequence()` at all if we have to use `_timescaledb_catalog.continuous_agg_migrate_plan_step_step_id_seq` anyway.

Reply:
Same here:

```sql
fabrizio=# ALTER EXTENSION timescaledb DROP SEQUENCE pg_get_serial_sequence('_timescaledb_catalog.continuous_agg_migrate_plan_step', 'step_id');
ERROR:  syntax error at or near "("
LINE 1: ...N timescaledb DROP SEQUENCE pg_get_serial_sequence('_timesca...
                                                             ^
```

Reply:
> We probably don't need to use pg_get_serial_sequence() at all 

That's why then we should be consistent, if we use `continuous_agg_migrate_plan_step_step_id_seq` we have to remove `pg_get_serial_sequence('_timescaledb_catalog.continuous_agg_migrate_plan_step', 'step_id')` from the code.

Reply:
Fixed